concent_api/core/payments/sci_backend.py

```diff
@@ -0,0 +1,93 @@
+import math
+from django.conf    import settings
+from golem_sci      import implementation
+
+
+def get_number_of_eth_block(concent_rpc: implementation.SCIImplementation):
+    assert isinstance(concent_rpc, implementation.SCIImplementation)
+    return concent_rpc.get_block_number()
+
+
+def get_list_of_payments(
+    concent_rpc:            implementation.SCIImplementation,
+    requestor_eth_address:  str,
+    provider_eth_address:   str,
+    payment_ts:             int,
+    current_time:           int,
+    to_block:               int,
+    transaction_type:       str,
+):
+    """
+    Function which return list of transactions from payment API
+    where timestamp >= T0
+    """
+    assert isinstance(concent_rpc,              implementation.SCIImplementation)
+    assert isinstance(requestor_eth_address,    str) and len(requestor_eth_address) == 42
+    assert isinstance(provider_eth_address,     str) and len(provider_eth_address)  == 42
+    assert isinstance(payment_ts,               int) and payment_ts     > 0
```

Comment:
Technically it is possible for both this time values to be `0`.

Reply:
Done